docs/en/ingest-management/release-notes/release-notes-8.13.asciidoc

```diff
@@ -14,13 +14,37 @@
 
 This section summarizes the changes in each release.
 
+* <<release-notes-8.13.1>>
 * <<release-notes-8.13.0>>
 
 Also see:
 
 * {kibana-ref}/release-notes.html[{kib} release notes]
 * {beats-ref}/release-notes.html[{beats} release notes]
 
+// begin 8.13.1 relnotes
+
+[[release-notes-8.13.1]]
+== {fleet} and {agent} 8.13.1
+
+Review important information about {fleet-server} and {agent} for the 8.13.1 release.
+
+[discrete]
+[[enhancements-8.13.1]]
+=== Enhancements
+
+{fleet}::
+* Feature/default fields ({kibana-pull}178020[#178020]).
```

Comment:
This needs a better name - the original PR didn't have a meaningful name. 

```
Remove `index.query.default_field` setting from managed component template settings.
```

cc @gsantoro to keep me honest on the impact of that PR

Reply:
I updated the text via [4daf32a](https://github.com/elastic/ingest-docs/pull/992/commits/4daf32a430b9f774b35ab6822520cf626f3690a6)